code/src/components/TodoItem.js

```diff
@@ -0,0 +1,142 @@
+import React, { useState } from 'react';
+import { useDispatch } from 'react-redux';
+import moment from 'moment';
+import styled from 'styled-components';
+
+import { todos } from '../reducers/todos';
+import CustomCheckbox from './CustomCheckbox';
+
+const TodoItem = ({ item }) => {
+  const [active, setActive] = useState('');
+
+  const dispatch = useDispatch();
+
+  const onRemoveTodo = () => {
+    dispatch(
+      todos.actions.removeTodo({
+        id: item.id,
+      })
+    );
+  };
+
+  const handleOnChange = event => {
+    dispatch(todos.actions.setComplete(item.id));
+  };
+
+  const toggleAccordion = () => {
+    setActive(active === '' ? 'active' : '');
+  };
```

Comment:
Love this! I didn't notice it at first - maybe add another cursor or a little arrow or something to visualize that it's possible to expand the todos?